### ISSUE TITLE
release:v6.2.0-DMR

### ISSUE THREAD

**Issue by Ivy-YinSu** [pingcap/tidb#35452]
### Candidate SHA: 
- TiDB: daf2b17cdfe30c02ce282361009fb5bdb05f2b0e
- TiKV: 4f717716a515a8047d91f4cbeef219f41c20dc2c
- PD: db465fee5548bf747b0a82b821d01c4c72b34b4f
- TiFlash: acd747f28bb8f48e00844427147e06a33cc2668a
- BR: daf2b17cdfe30c02ce282361009fb5bdb05f2b0e
- binlog: facf8f94aede062cbf57d9f7ac0c2ec364fd6855
- tidb-tools: b505953aaa9ed289b71f3000cb7cb54fd09cd3e2
- tiflow: c0eaa0075d46e3011225796c7f060e2a5d903a77
- dumpling: daf2b17cdfe30c02ce282361009fb5bdb05f2b0e

Working Tags: v6.2.0

### Check the diff in the current version:
[TiDB diff](https://github.com/pingcap/tidb/compare/v6.1.0...master)
[TiKV diff](https://github.com/tikv/tikv/compare/v6.1.0...master)
[PD diff](https://github.com/tikv/pd/compare/v6.1.0...master)
Tools:
[BR diff](https://github.com/pingcap/br/compare/v6.1.0...master)
[Binlog diff](https://github.com/pingcap/tidb-binlog/compare/v6.1.0...master)
[importer diff](https://github.com/tikv/importer/compare/v6.1.0...master)
[tidb-tools diff](https://github.com/pingcap/tidb-tools/compare/v6.1.0...master)
[ticdc diff](https://github.com/pingcap/ticdc/compare/v6.1.0...master)
[dumpling diff](https://github.com/pingcap/dumpling/compare/v6.1.0...master)

-------------------------------
### Release process checklist

- [x] 0. Announce the release plan
- [x] 1. Code freeze: Jul. 21st
- [x] 2. Build finish: Jul. 22nd
- [x] 3. Qualify the Release. 
- [x] 4. Doc prepared for Release.
 
**Release date: Aug. 23th**
Do not proceed below until the release date.

- [x] 5. Publish TiDB Release 
> - [x] fill in Candidate SHA above
> - [x] fill in release tag
- [x] 6. Update docs (handled by Docs team)
- [x] 7. Update the offical website

**Comment by Ivy-YinSu** [comment]
Hello Everyone,
We are sorry to announce that the TiDB v6.2.0-DMR release will be postponed from 08/11 to 08/16. 

Because of the following critical issues may cause a quality risk. 
https://github.com/pingcap/tidb/issues/36955
We treat every quality issue with care, so we will take several days to fix the issue and go through the regression test.

If this change causes you any trouble, please feel free to contact us via [release@pingcap.com](mailto:release@pingcap.com).

Thanks,
Release team
